pkg/wkspace/builder.go

```diff
@@ -198,6 +200,9 @@ func DiffedRepos() ([]string, error) {
 		result[count] = repo
 		count++
 	}
+
+	fmt.Printf(" files %s    repos %s \n", files, repos)
```

Comment:
@dustinfaulkner91 This is one of the extra debug statements that should be removed.